src/main/java/org/blackdread/sqltojava/util/SqlUtils.java

```diff
@@ -53,7 +55,10 @@ public static Optional<Integer> parseSqlSize(String value) {
     }
 
     public static String parseSqlType(String value) {
-        String typeName = value.split("\\(")[0];
-        return typeName;
+        // todo: Oracle Bigint, BigDecimal
```

Comment:
I am not familiar with orcale, can you give a brief explanation what is the specific case here?

Reply:
Oracle does not support different number types like bigint, tinyint, bigdecimal etc. All are represented by the `NUMBER` type. I need to know what length of field for number type. The `parseSqlType` function removes the length so I added extra condition.

Reply:
Alright, as far as I could check in this code, seems that `NUMBER` is specific to oracle, it does not make other DBs logic fail or have different logic result in JDL, etc.
If I have time, I will read oracle docs and try to come up with solution if any.

Are the following conventions within oracle DB to consider?
- `NUMBER(38)` -> a long
- `NUMBER(19,5)` -> a decimal

What if someones uses `NUMBER(20, 6)` ? It seems that current logic would in a `int` for generated JDL of `oracle`, am I right?